docs/README.rst

```diff
@@ -1,41 +1,170 @@
-tractor
-=======
-A `structured concurrent`_, async-native "`actor model`_" built on trio_ and multiprocessing_.
+|logo| ``tractor``: next-gen Python parallelism
 
 |gh_actions|
 |docs|
 
 .. _actor model: https://en.wikipedia.org/wiki/Actor_model
 .. _trio: https://github.com/python-trio/trio
-.. _multiprocessing: https://en.wikipedia.org/wiki/Multiprocessing
+.. _multi-processing: https://en.wikipedia.org/wiki/Multiprocessing
 .. _trionic: https://trio.readthedocs.io/en/latest/design.html#high-level-design-principles
 .. _async sandwich: https://trio.readthedocs.io/en/latest/tutorial.html#async-sandwich
 .. _structured concurrent: https://trio.discourse.group/t/concise-definition-of-structured-concurrency/228
 
 
-``tractor`` is an attempt to bring trionic_ `structured concurrency`_ to
-distributed multi-core Python; it aims to be the Python multi-processing
-framework *you always wanted*.
+``tractor`` is a `structured concurrent`_ "`actor model`_" built on trio_ and multi-processing_.
 
-``tractor`` lets you spawn ``trio`` *"actors"*: processes which each run
-a ``trio`` scheduled task tree (also known as an `async sandwich`_).
-*Actors* communicate by exchanging asynchronous messages_ and avoid
-sharing any state. This model allows for highly distributed software
-architecture which works just as well on multiple cores as it does over
-many hosts.
+We pair structured concurrency with true multi-core Python parallelism;
+the aim is to be the multi-processing framework *you always wanted*.
 
-The first step to grok ``tractor`` is to get the basics of ``trio`` down.
+The first steps to grok ``tractor`` is to get the basics of ``trio`` down.
 A great place to start is the `trio docs`_ and this `blog post`_.
 
-.. _messages: https://en.wikipedia.org/wiki/Message_passing
-.. _trio docs: https://trio.readthedocs.io/en/latest/
-.. _blog post: https://vorpus.org/blog/notes-on-structured-concurrency-or-go-statement-considered-harmful/
-.. _structured concurrency: https://vorpus.org/blog/notes-on-structured-concurrency-or-go-statement-considered-harmful/
-.. _3 axioms: https://en.wikipedia.org/wiki/Actor_model#Fundamental_concepts
-.. _unrequirements: https://en.wikipedia.org/wiki/Actor_model#Direct_communication_and_asynchrony
-.. _async generators: https://www.python.org/dev/peps/pep-0525/
+
+Features
+--------
+- **It's just** a ``trio`` API
+- Infinitely nesteable process trees
+- Built-in APIs for inter-process streaming
+- A (first ever?) "native" multi-core debugger for Python using `pdb++`_
+- (Soon to land) ``asyncio`` support allowing for "infected" actors where
+  `trio` drives the `asyncio` scheduler via the astounding "`guest mode`_"
+- (Soon to land) typed messaging protocols (ex. via ``msgspec``)
+
+
+Zombie safe: self-destruct a process tree
+-----------------------------------------
+``tractor`` (attempts to) protect from zombies, no matter what.
+
+.. code:: python
+
+    """
```

Comment:
Per the reST specs,

> Several constructs begin with a marker, and the body of the construct
> must be indented relative to the marker.

where in this case, the `code-block` (I can't seem to find `code` even in Sphinx-flavored reST) body should be indented by 3 space.  I believe that the editor picked this up and caused some indentation issues below.

Reply:
Oh so you mean *after` the `..` there should be 3 spaces before the `code::`?

Reply:
Yep, e.g.

```rst
.. code-block:: python

   """
   ...
   async def target():
       try:
   ...
```

BTW could you please hide the resolved reviews, my international connection is terrible due to cable maintenance or something and GitHub's way of fetching content really does not help :smile: